src/Servers/HttpSys/src/RequestProcessing/RequestContext.cs

```diff
@@ -254,7 +254,6 @@ protected void SetFatalResponse(int status)
     {
         Response.StatusCode = status;
         Response.ContentLength = 0;
-        Dispose();
```

Comment:
Is this called from other places that are now not disposing the context?

Reply:
Yes, it is now called explicitly in all other places:
- https://github.com/dotnet/aspnetcore/pull/42223/files#diff-a78e961f758bf8f4f795410ae852718a6d31dec5599d398f12ffd4c58034d33eR35
- https://github.com/dotnet/aspnetcore/pull/42223/files#diff-a78e961f758bf8f4f795410ae852718a6d31dec5599d398f12ffd4c58034d33eR99